src/Main.php

```diff
@@ -150,6 +150,7 @@ function showUsageAndExit()
         $gapicYaml,
         $serviceYaml,
         $numericEnums,
+        -1,
```

Comment:
Sorry, what is this for?

Reply:
Oh ok...maybe this is a "nit" but could you add `/* $licenseYear */` or something to this so that we know what it is? Probably not a big deal, just looked out of place in the context of this CL.

Reply:
Sure thing